src/components/RequestDashboard/styles.jsx

```diff
@@ -0,0 +1,149 @@
+import { makeStyles } from '@mui/styles';
+export default makeStyles(
+  theme => ({
+    disappear: {
+      display: 'none'
+    },
+    spacer: {
+      height: '50px', // must be same as buttons
+      borderBottom: '1px solid black',
+      flexGrow: 1,
+      backgroundColor: '#005B94'
+    },
+    mainButton: {
+      '&.MuiButtonBase-root': {
+        // transition: 'all 250ms ease-in 0ms',
+      }
+    },
+    mainButtonView: {
+      '&.MuiButtonBase-root': {
+        width: '600px',
+        maxWidth: '90%',
+        height: '150px',
+        backgroundColor: '#0d6efd',
+        opacity: '75%',
+        color: '#fcfcfc',
+        fontSize: '1.5rem'
+      }
+    },
+    mainDiv: {},
+    mainDivView: {
+      '&.MuiGrid-root': {
+        padding: '0 50px 0 50px',
+        marginTop: '35vh'
+      }
+    },
+    mainIcon: {
+      '&.MuiSvgIcon-root': {
+        '&.MuiSvgIcon-fontSizeMedium': {
+          fontSize: '3rem'
+        }
+      }
+    },
+    mainSectionView: {
+      width: 'auto',
+      height: 'auto',
+      // margin: '0 15px 0 15px', // must be same as tabDivView
+      borderLeft: '1px solid black',
+      borderRight: '1px solid black'
+    },
+    noTasks: {
+      backgroundColor: '#e4e4e4',
+      padding: '10px',
+      fontSize: '18px'
+    },
+    // DO NOT ALPHABETIZE
+    // if you must alphabetize this file to have classes
+    // sorted, rename tabButtonView and selectedTabView such
+    // that tabButtonView occurs earlier in the list.
+    // Otherwise, the styles will override incorrectly.
+    tabButtonView: {
+      '&.MuiButtonBase-root': {
+        width: '75px',
+        height: '50px',
+        opacity: '75%',
+        fontSize: '1.5rem',
+        border: '1px solid black',
+        boxShadow: 'none',
+        borderRadius: '0'
+      },
+      '& > *': {
+        // generic child selector
+        '&.MuiButton-iconSizeMedium': {
+          // specificty
```

Comment:
```suggestion
          // specificity
```
good job figuring out the MUI 5 pseudoclasses